demo/tree-view-demo.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { TreeView } from '../src';
+import { TreeItemData } from '../src/components/tree-view/tree-view';
+
+interface TreeViewDemoState {
+    selectedItem: Object;
+}
+
+export const treeData: TreeItemData[] = [
```

Comment:
Please make the demo more accessible to us humans. It can be more tree-ish. Yes, one level of children is technically a tree. BUT. Let your creative self unhinge itself for the common good. 'label0'? We humans look for meaning. Draw inspiration from the beautiful world around us! Add some layer to your tree! Let it bloom!